frontend/packages/app/src/clutch.config.js

```diff
@@ -46,4 +46,41 @@ module.exports = {
       },
     },
   },
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      trending: true,
+      componentProps: {
+        columns: ["identifier", "targets", "type", "description"],
+        experimentTypes: {
+          "type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestSpecification": {
+            mapping: e => {
```

Comment:
This is still a function -> not ideal for a config file.

Reply:
I wanted to move the implementation of this function to another file and import it here but I ran into a few issues. 

First, nothing from outside of `/src` directory can be imported in here which means that I could move the implementation of the function to `serverexperimentation` package:

<img width="1292" alt="Screen Shot 2020-08-11 at 4 29 47 PM" src="https://user-images.githubusercontent.com/4410346/90083961-d39be280-dcc8-11ea-847f-4efc69efbf36.png">

Secondly, even If I move the implementation of the function to the file that lives inside of the `/src` directory (again, this limitation makes this far from being perfect) I see the following error once I add an import statement at the top of the file:

<img width="1045" alt="Screen Shot 2020-08-11 at 4 37 53 PM" src="https://user-images.githubusercontent.com/4410346/90083992-f4fcce80-dcc8-11ea-8b43-c60d1893f0f2.png">